classes/announcement/Command.inc.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file clases/announcement/Command.inc.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class announcement
+ *
+ * @brief Commands for modifying announcements
+ */
+
+namespace PKP\Announcement;
```

Comment:
The namespace needs to be the same case as the pathname the class is in, for PSR-4 compatibility -- `namespace PKP\announcement;`